rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -91,6 +91,16 @@ object Jupyter {
                             Some(xbounds))
     })
 
+  def whiskers[K, N](samples: Seq[Map[String, Double]]): Plot = {
+    val labels = samples.head.keys.toList.sorted
+    val data = labels.map { k =>
+      samples.map { m =>
+        m(k)
+      }
+    }
+    BoxPlot(data).standard(xLabels = labels)
+  }
```

Comment:
I'd suggest implementing this form as well and having the current form call this:
```
def whiskers(data: Map[String, Seq[Double]]): Plot =
  BoxPlot(data.values).standard(xLabels = data.keys)
```